Building on x86_64-musl fails with:

	TerminalView.m:49:12: fatal error: termio.h: No such file or directory

--- TerminalView.m        2017-08-02 13:17:43.000000000 -0400
+++ TerminalView.m        2020-12-17 01:38:16.124286870 -0500
@@ -49 +49 @@ activated */
-#  include <termio.h>
+#  include <termios.h>
